invenio_app_ils/records/loaders/schemas/identifiers.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Identifiers schema for marshmallow loader."""
+
+from marshmallow import EXCLUDE, Schema, fields
+
+
+class IdentifiersSchema(Schema):
```

Comment:
this schema already exists under document.py, it would be better to reuse it than create a duplicate